torch/_torch_docs.py

```diff
@@ -8833,6 +8839,39 @@ def merge_dicts(*dicts):
     tensor([ 4,  5,  8])
 """.format(**common_args))
 
+add_docstr(torch.take_along_dim,
+           r"""
+take_along_dim(input, indices, dim, *, out=None) -> Tensor
+
+Selects values from :attr:`input` at the 1-dimensional indices from :attr:`indices` along the given :attr:`dim`.
+
+Functions returning index along dimension like torch.argmax and torch.argsort can be passed
```

Comment:
"Functions that return indices along a dimension, like :func:\`torch.argmax\` and :func:\`torch.argsort\`, are designed to work with this function. See the examples below."

Make sure the function references work as expected.